kornia/filters/__tmp__.py

```diff
@@ -2,8 +2,6 @@
 from functools import wraps
 from typing import Callable
 
-warnings.simplefilter('always', DeprecationWarning)
```

Comment:
This can be put inside the two wrappers below. It will only be activated if a deprecated function is used.

Reply:
It'll definitely solve my issue as I don't call the deprecated functions, so I'm pretty fine with that solution.
But I'm still a bit wondering that would it be really OK to change the warning behavior globally even inside a function? Because I concern that users may still be a bit confused if other unrelated packages suddenly start to show lots of warning after running a certain code. It's much better than happening when _importing_ kornia though, because what has triggered that is more clear.

If you consider these deprecation should be treated seriously enough to make sure to show on the screen by default, maybe it's higher than python's DeprecationWarning is originally intended for. How about replacing warning type by `UserWarning` or `FutureWarning` that won't be ignored? (numpy seems to partially use `FutureWarning` for this purpose, though it's not thoroughly done)
https://docs.python.org/3/library/warnings.html#warning-categories

Reply:
definitely - `FutureWarning` seems a more suitable category for our use case

Reply:
I have updated the PR in that way. I'm still open to any discussions!